src/rebuild/scan.c

```diff
@@ -839,6 +839,17 @@ rebuild_container_scan_cb(daos_handle_t ih, vos_iter_entry_t *entry,
 			DP_UUID(entry->ie_couuid), DP_RC(rc));
 		D_GOTO(close, rc);
 	}
+
+	/*
+	 * The container may has been closed by the application, but some resource (DRAM) occupied
+	 * by DTX may be not released because DTX resync was in-progress at that time. When arrive
+	 * here, DTX resync must has completed globally. Let's release related resource.
+	 */
+	if (unlikely(cont_child->sc_dtx_delay_reset == 1)) {
```

Comment:
seems the DTX resync possibly not caused by rebuild? (DTX resync may start, and no rebuild happended).
if the rebuild not started or even disabled, which part can do this kind of resource freeing?

Reply:
If the DTX resync is triggered by opening container, then the DTX table cannot be released since the opened container will use it. Related resource will be released via DTX aggregation step by step.